doc/classes/Node.xml

```diff
@@ -101,7 +101,8 @@
 				Called when an [InputEvent] hasn't been consumed by [method _input] or any GUI. The input event propagates up through the node tree until a node consumes it.
```

Comment:
I don't think the right language is used here. The call doesn't actually "propagate" from what I can tell (see `SceneTree::_call_input_pause`). 

> The input event propagates up through the node tree

This language might mislead the user into thinking the call is repeated on parents until it is handled, or until the last parent. This is not correct. `call_input_pause` only calls the method on the node it is registered to call on. For example, MenuButtons are registered to receive `unhandled_key_input`. The `_unhandled_key_input` method is only called on the MenuButton - not parents, and not children.

I would probably change this to something more accurate:
"`_unhandled_input` is called on every node which is registered to receive it until one of the nodes handles the input."

However this relies on the information that "It is only called if unhandled input processing is enabled" so this might be better to place after the next line
The usage of the word "propagates" may also mislead the user into thinking at `Object::propagate_call` is used behind the scenes for this functionality.

Reply:
P.S. I know this isn't part of the PR but I thought since we are here, let's discuss

Reply:
You are right on all the points, but to be honest I don't think we should care that much.
The function is indeed not called on all the nodes, as they indeed have to implement the method. I am not sure it is worth mentioning to the user, as they probably know know an unexisting function cannot be called.

Regarding the "propagate" term, in the end it's the same. It's a similar behavior to calling `Node::propagate_call`, as the node order produced by `Node::is_greater_than` will lead to the same order as what a reversed propagate_call would do. That's why the documentation mentions "propagates up through the node tree".

We have to write the documentation from the user point of view, what happen behind the scene for performance reasons (like adding the nodes to a group, so that the function is only called on specific nodes, etc...) is not important. What matters is the end result.